std-ddox.ddoc

```diff
@@ -3,6 +3,8 @@ XREF = std.$1.$2
 XREF_PACK = std.$1.$2.$3
 CXREF = core.$1.$2
 ECXREF = etc.$1.$2
+MREF_HELPER=/$1$(MREF_HELPER $+)
+MREF_HELPER2=.$1$(MREF_HELPER2 $+)
```

Comment:
Per my comment in the other PR, these would be `SLASH_PREFIXED` and `DOT_PREFIXED`.